Pet-Clinic-Frontend/src/pages/Landing_page.tsx

```diff
@@ -0,0 +1,15 @@
+import './Landing_page.css';
+import clinicImage from "../images/petclinic.png";
+
+function App() {
```

Comment:
perhaps a more descriptive function name? On the other hand, this also works

Reply:
Yeah maybe LandingPage?